CHANGELOG.md

```diff
@@ -2,7 +2,42 @@
 All notable changes to this project will be documented in this file.
 See the [change log guidelines](http://keepachangelog.com/) for information on how to structure the file.
 
-## [Unreleased]
+## [5.1.0]
```

Comment:
The link here would be `https://github.com/halestudio/hale/compare/5.0.1...5.1.0`

The `Unreleased` link also needs to be updated.

Reply:
FYI Currently the links are defined [at the end of the document](https://github.com/halestudio/hale/blob/bb04e1dd0e27209353024536837b47e1c3987e0b/CHANGELOG.md?plain=1#L452).

Reply:
Thank you. I'll add also the missing versions.